### DISCUSSION TITLE
[ARCHIVED] Museum project - Wireframes & planning

### DISCUSSION THREAD

**Discussion by periode** [periode/augmenting-gallery#13] [Homework]
- Finish a first pass on your wireframe for your museum project; it should be fleshed out enough to allow your classmates and instructors to give meaningful feedback. Come ready to present it in class.

- Establish a retroplanning for your project, breaking down the next three weeks at the day level, and post it below (it can be a google spreadsheet, for instance).

**Comment by aashnasoni** [comment]
Planning for project: https://docs.google.com/spreadsheets/d/1r-uub3LNh87is-0gBfPq1i6j2Z1-It-DUPCg_CjVmaw/edit?usp=sharing
Wireframe: https://www.figma.com/file/HPR0BXyLvnJEzC7fXjP60M/Wireframe-for-Adventure-Game?node-id=0%3A1

**Reply by periode** [reply]
I can't access the planning for the project due to sharing permissions, but here are some points about the current state of your project:

- you still need to find more precise phrasing and narrative framing for finding the objects
- to deepen this further, you should use at least 3 or 4 objects to show how they could connect to one another
- finally, you should also consider giving concrete examples of what the rewards would be (is it only from the gift shop? could it also be tickets to events/concerts at the HF? could it be exclusively digital?)

in general, the overall idea is there, but does need further elaboration with concrete examples.

**Comment by miafan716** [comment]
Valentina and Mia, Adventure Game, Humboldt Forum
[Retroplanning](https://docs.google.com/spreadsheets/d/119qubvTj0qL9AEp1gsdHSy2GHa8DCz3xT51GMFGfTV0/edit?usp=sharing)
[Wireframe](https://www.figma.com/file/WKIcog3jA7QlJ0VwugiLqe/HF-Adventure-Game-Valentina%26Mia?node-id=0%3A1)

**Reply by periode** [reply]
The wireframe is clear and the planning looks good! The two main aspects you should focus on now are:
- finding a narrative thread for the adventure game
- developping the interaction model with the virtual guide: scan an image in unity to have a guide appear and either lead towards the next statue or discuss the current statue.

In terms of planning, it looks like you're on track: this week should be focusing on Unity (but do not wait to have final assets to have a functioning prototype)

**Comment by yylinj** [comment]
Nika and Yilin, Get Creative, Humboldt Forum

Wireframe: https://www.figma.com/file/ImgLGHyJPviyXk4oTKzkMJ/humboldt?node-id=0%3A1
Retroplanning: https://docs.google.com/spreadsheets/d/1xXlYdYR1UoillRJij5wEtHjlXN12xCpGBlcADoutEmU/edit?usp=sharing

**Reply by periode** [reply]
Once you guys clarify the relation between 3D-mode, on-site mode, AR-mode and off-site mode, your wireframe will be quite solid.

In terms of planning, though, I would be allocate more time building a simple prototype in Unity to manipulate 3D models, to inform further your decision to split the experience between 3D and AR.

**Comment by brianajpierre** [comment]
Retroplanning: https://docs.google.com/document/d/1Be-Lmh6gfbUdNE_LHjLe9HaW_g3LhMq9ZDNiHdfNHD0/edit?usp=sharing
(Bottom of Google Doc)
Wireframe: https://www.figma.com/file/1660VL5OPBZlCgisgHJ70H/WireFrame-1?node-id=0%3A1

**Reply by chloeumlaut** [reply]
https://docs.google.com/spreadsheets/d/1Rzin085nPu8B5Qh9mkfaa1hxD9fBKudn2ZjD92_TBc4/edit?usp=sharing

**Reply by brianajpierre** [reply]
New link: https://www.figma.com/file/1660VL5OPBZlCgisgHJ70H/WireFrame-1?node-id=0%3A1

**Reply by periode** [reply]
Given the new deadline (May 11th, you now have one week more!), keep in mind that you must either develop 1 wireframe and 1 unity prototype, or 2 wireframes, which develop 2 different aspects to your concept.

For instance, you can develop one version where you examine the trajectory of the object through a map interface, recreating Jacobsen's perspective as a world traveller, or through a more subjective interface (perhaps looking through the object's eye?).

In terms of content, then, you should focus on what are all the parameters and options that you can give to the visitor in terms of choosing where the object should go, in order to make this choice meaningful.

**Comment by joshblecherman** [comment]
[Wireframe](https://www.figma.com/file/HPqN50mGVo9EZNIWSGRoZU/Future-Walk-Wireframe?node-id=0%3A1)
[Retroplanning](https://docs.google.com/spreadsheets/d/18cbqU-H18iq3HZLpAkVwCb5KcaQcLdSLemCm3UzDCUk/edit?usp=sharing)

**Reply by periode** [reply]
It looks like you're a bit late on your planning regarding fleshing out the wireframe (particularly in terms of placholder text, and in terms of the conclusion of the experience). Additionally, you should focus the last week not just on revisiting the wireframing, but also developping a Unity prototype to communicate the interaction of picking up pieces and assembling a harpsichord.

**Comment by mdl601** [comment]
Wireframe: https://www.figma.com/file/1VM8KAuDDXT4Zh9D53DVrd/Jacobsen-Gallery-Wireframe?node-id=0%3A1
Retroplaning: https://docs.google.com/spreadsheets/d/1ua0dIHKWaS8hVHEJaBWeXZs4Rh0nzRLImHlI_L9rw9A/edit?usp=sharing

**Reply by periode** [reply]
The wireframe looks good and it seems like you are on track, once you've added the time to either develop a unity prototype, or design an second figma wireframe.

As for the wireframe, I see three points for improvement:
- doing more research to identify other places where the chosen object has been
- provide more of a narrative direction: it's not clear why the visitor should start the experience to begin with, and how it ends
- provide less "monolithic" ways to communicate information about the objects: are one big block of text and one big block of audio recordings the only way to explore the object's different environments?

**Comment by Lkochar19** [comment]
[Wireframe](https://www.figma.com/file/vx8AwJh0FiGiGijJTIgMvS/HFProject?node-id=0%3A1)
[Retroplanning](https://docs.google.com/spreadsheets/d/1ixsBqk-aX1VWGjLDCqPpRswdxOO3hzWX3OJd0nny4I0/edit?usp=sharing)

**Reply by periode** [reply]
The planning doesn't seem to include the most important part of the project: visualizing how one "re-organizes" the different eras of the HF, which would be particularly effective if it was developed in Unity rather than in a static wireframe (which, in the current state of the three screenshots, is still of limited use).

Additionally, you have to provide a conclusion to the experience: what happens when the buildings are re-organized? How do you wrap-up the evperience?

**Comment by IanPort** [comment]
Ian, Maria, Dove
[First wireframe of Future Walk addition](https://www.figma.com/file/98OaPeZgmev9zDPpLYPS66/Future-Walk---Harpsichord?node-id=0%3A1&t=OtnESihdqWpoalyH-1)
Sketches not yet digitalized:
![tempImagebWDvUB](https://user-images.githubusercontent.com/113472376/204666640-14b3c8c5-2874-4673-8d10-3f24f1915539.gif)

**Comment by olivestew222** [comment]
wireframe adventure game (alaya, olive, hubert)- https://www.figma.com/file/PCOY6mjc69owaxWDmYxyDx/Untitled?node-id=0%3A1&t=PutDhQ4XWbfWzgVD-0

**Comment by kylewenchang** [comment]
[Wireframe - Kyle, Vasant, Kendall](https://www.figma.com/file/ymObwFwoxhgZsNrFmp3g2q/Untitled?node-id=0%3A1&t=YdUTB8mU8tj0HhuR-1)

**Comment by Dav1dyang** [comment]
Wireframe 1: https://www.figma.com/file/3A9fJR7evoU9vEDe4Iciul/Humboldt-Wireframe-Color-and-AR?node-id=0%3A1&t=wKyduczbIG2kaHc2-1
Wireframe 2: https://www.figma.com/file/en1mSdc6QZt7E6dA2DVh3Q/Humboldt-Architecture?node-id=0%3A1&t=wonZjdnFUfoe4S6m-1
